pylintrc

```diff
@@ -96,7 +93,6 @@ enable=signature-differs,
        print-function,
        unittest-assert-raises,
        pytest-raises-multiple-statements,
-       use-set-literal,
```

Comment:
after removing this, is `pylint_plugins/set_checker.py` still needed?

Reply:
No, it's not. We can remove it in a follow-up.

Reply:
@gabrielfu Filed #8925 :)